modules/serverless-rn-1-9-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-9-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.9.0
+
+* {ServerlessProductName} now uses Knative Serving 0.15.0.
+* {ServerlessProductName} now uses Knative Serving Operator 0.15.0.
```

Comment:
The Knative Serving and Knative Eventing operators (a few lines below) are now merged into a common "Knative Operator 0.15.2"